app/templates/search/_services_save_search.html

```diff
@@ -1,15 +1,21 @@
+{# this file can be used from contexts that don't inherit from _base_page.html, so we must ensure we have all imports we need #}
+{% from "components/button/macro.njk" import govukButton %}
```

Comment:
😭, thanks for documenting this partial and explaining why an import was needed here.